.gitignore

```diff
@@ -1,11 +1,13 @@
-.env
-.DS_Store
 **/.observablehq/cache/
-coverage/
-docs/themes.md
-docs/theme/*.md
-dist/
+.DS_Store
+.env
+/build.test/
+/build/
```

Comment:
The leading slash here means only ignore `build/` at the project root, not `test/input/build/` etc.